src/store/route3/route3-state.ts

```diff
@@ -0,0 +1,20 @@
+import { Route3Dex, Route3SwapParamsResponse, Route3Token } from 'src/interfaces/route3.interface';
+
+import { createEntity } from '../create-entity';
+import { LoadableEntityState } from '../types';
+
+export interface Route3State {
```

Comment:
the same https://github.com/madfish-solutions/templewallet-extension/pull/854#discussion_r1109918111